Assets/MRTK/Tests/PlayModeTests/GltfTests.cs

```diff
@@ -98,8 +100,32 @@ public IEnumerator TestGltfCustomAttributes()
             int temperature = gltfObject.accessors[temperatureIdx].count;
             Assert.AreEqual(100, temperature);
         }
+        
+        [UnityTest]
+        public IEnumerator TestGltfCustomAttributesData() {
+            // Load glTF
+            string path = AssetDatabase.GUIDToAssetPath(CubeCustomAttrGuid);
+            var task = GltfUtility.ImportGltfObjectFromPathAsync(path);
+
+            yield return WaitForTask(task);
+
+            GltfObject gltfObject = task.Result;
+
+            yield return null;
+
+            // Check for custom vertex data is a list of 10s
+            gltfObject.meshes[0].primitives[0].Attributes.TryGetValue("_CUSTOM_ATTR", out var customAttrIdx);
+
+            GltfAccessor accessor = gltfObject.GetAccessor(customAttrIdx);
+            var intArray = accessor.GetIntArray(false);
+
+            foreach (var item in intArray)
+            {
+                Assert.AreEqual(10, item);
+            }
+        }
         #endregion
         
-    }
+    } 
```

Comment:
nit: extra whitespace

```suggestion
    }
```